punishments.js

```diff
@@ -171,6 +172,7 @@ Punishments.punishmentTypes = new Map([
 	['LOCK', 'locked'],
 	['BAN', 'globally banned'],
 	['NAMELOCK', 'namelocked'],
+	['BATTLEBAN', 'battleBanned'],
```

Comment:
Why camel case? see global ban casing here

Reply:
Personal habit I figured someone was going to point it but only realized after finishing.